docs/workflow/debugging/coreclr/debugging.md

```diff
@@ -19,10 +19,16 @@ Debugging CoreCLR on Windows
 7. Set Command Arguments=`<managed app you wish to run>` (e.g. HelloWorld.dll)
 8. Set Working Directory=`$(SolutionDir)\..\..\..\bin\coreclr\Windows_NT.$(Platform).$(Configuration)`
     1. This points to the folder containing CoreCLR binaries.
-9. Press F11 to start debugging at wmain in corerun (or set a breakpoint in source and press F5 to run to it)
+9. Set Environment=`CORE_LIBRARIES=$(SolutionDir)\..\..\..\bin\runtime\<current tfm>-Windows_NT-$(Configuration)-$(Platform)`,
```

Comment:
A nit - it might be good to mention that this is needed only in case when you are debugging coreclr using an "external" application. You don't need to do this when debugging it using coreclr tests.